plugins/inputs/execd/sample.conf

```
@@ -23,6 +23,10 @@
   ## Delay before the process is restarted after an unexpected termination
   restart_delay = "10s"
 
+  ## Buffer size used to read from the command output stream
+  ## Optional parameter. Default is 64 Kib, minimum is 16 bytes
+  buffer_size = "64Kib"
```

As this is a sensible default and also set in `init()` I think you should comment that option
```suggestion
  # buffer_size = "64Kib"
```

---

Thanks for the suggestion!
I'm not sure I understand correctly: should I remove the option completely or just remove it from sample.conf?

---

all you need to do is accept Sven's suggestion. 

Essentially, default values should be commented out in the config. We still want it in the config so the user sees it, but commented out to indicate it is not actually required or a user can un-comment it and set their own value.